jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -474,7 +478,8 @@ public void create(boolean waitFor) {
         // NOTE: this is required for Mac OS X!
         mainThread = Thread.currentThread();
         mainThread.setName("jME3 Main");
-        run();
+        new Thread(this, "jME3 Main").start();
```

Comment:
These changes seem unrelated to the issue addressed by this PR.

Reply:
NOt sure how to stop that.  it automatically pulled them in from my fork.  Do you know how to limit what changed to pull in.